src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -103,7 +103,11 @@ public HttpMethod Method
             {
                 if ((value != null) && (value.IsAbsoluteUri) && (!HttpUtilities.IsHttpUri(value)))
                 {
+#if TARGETS_BROWSER
```

Comment:
Ditto

Reply:
I like the more detailed one so left it in but moved the message definition to partial file as described.